specification/datashare/resource-manager/Microsoft.DataShare/stable/2020-09-01/DataShare.json

```diff
@@ -0,0 +1,6751 @@
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "$skipToken",
+            "in": "query",
+            "description": "Continuation token",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "Filters the results using OData syntax.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$orderby",
+            "in": "query",
+            "description": "Sorts the results using OData syntax.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ShareSynchronizationList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Microsoft.DataShare resource provider.",
+            "schema": {
+              "$ref": "#/definitions/DataShareError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Shares_ListSynchronizations": {
+            "$ref": "./examples/Shares_ListSynchronizations.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataShare/accounts/{accountName}/shares/{shareName}/providerShareSubscriptions/{providerShareSubscriptionId}/adjust": {
+      "post": {
+        "tags": [
+          "Share"
+        ],
+        "summary": "Adjust the expiration date of a share subscription in a provider share.",
+        "description": "Adjust a share subscription's expiration date in a provider share",
+        "operationId": "ProviderShareSubscriptions_Adjust",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/accountName"
+          },
+          {
+            "name": "shareName",
+            "in": "path",
+            "description": "The name of the share.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "providerShareSubscriptionId",
+            "in": "path",
+            "description": "To locate shareSubscription",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "providerShareSubscription",
+            "in": "body",
+            "description": "The provider share subscription",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/ProviderShareSubscription"
```

Comment:
Please modify the examples to show what this parameter is used for in adjust and reinstate.  Its not clear when/how this would be used if it is not required.  Its also odd that it takes the entire providerShareSubscription when it is only changing 1 property.

Reply:
@pilor  updated.